src/utils/endpoints.js

```diff
@@ -1,16 +1,19 @@
 export const endpoints = (operationId, param) => {
-  const baseUrl = 'http://localhost:3001/api';
+  const baseUrl = 'http://localhost:3001';
+  const apiBaseUrl = 'http://localhost:3001/api';
```

Comment:
I suggest to use baseURL: `apiBaseUrl = baseUrl + '/api'`

Reply:
Agree